pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -315,6 +314,13 @@ class Build {
         List dynamicList = buildConfig.DYNAMIC_LIST
         List numMachines = buildConfig.NUM_MACHINES
         def enableTestDynamicParallel = Boolean.valueOf(buildConfig.ENABLE_TESTDYNAMICPARALLEL)
+        def aqaBranch = "master"
+        def useTestEnvProperties = false
+        if (buildConfig.SCM_REF && buildConfig.AQA_REF) {
+            // We keep the ADOPTOPENJDK label for historical compatibility.
```

Comment:
do not think this suggested comment is needed anymore here

Reply:
Correct, can be removed.